docs/operations/high-availability.md

```diff
@@ -29,8 +29,7 @@ Apache ZooKeeper, metadata store, the coordinator, the overlord, and brokers are
 We recommend either installing ZooKeeper on its own hardware, or running 3 or 5 Master servers (where overlords or coordinators are running)
 and configuring ZooKeeper on them appropriately. See the [ZooKeeper admin guide](https://zookeeper.apache.org/doc/current/zookeeperAdmin) for more details.
 - For highly-available metadata storage, we recommend MySQL or PostgreSQL with replication and failover enabled.
-See [MySQL HA/Scalability Guide](https://dev.mysql.com/doc/mysql-ha-scalability/en/)
-and [PostgreSQL's High Availability, Load Balancing, and Replication](https://www.postgresql.org/docs/current/high-availability.html) for MySQL and PostgreSQL, respectively.
+See the "MySQL HA/Scalability Guide" and [PostgreSQL's High Availability, Load Balancing, and Replication](https://www.postgresql.org/docs/current/high-availability.html) for MySQL and PostgreSQL, respectively.
```

Comment:
since the first link is missing, maybe we should rephrase to not have the title of the linked article ("MySQL HA/Scalability Guide")

Reply:
I think for MySQL we can link to https://www.mysql.com/products/enterprise/high_availability.html.